tests/wrappers/core_wrappers_test.py

```diff
@@ -255,15 +284,13 @@ def observation(self, observation):
 
 
 def test_wrapped_observation_missing_definition(env: LlvmEnv):
-
-    env = ObservationWrapper(env)
-    with pytest.raises(NotImplementedError):
-        env.reset()
+    with pytest.raises(TypeError):
+        env = ObservationWrapper(env)
```

Comment:
Hmm, why is it now a `TypeError`? Could you add a `match="..."` argument to match a bit of the description?